drivers/sx127x/sx127x_netdev.c

```diff
@@ -559,6 +559,8 @@ static int _get_state(sx127x_t *dev, void *val)
             break;
 
         default:
+            /* prevent "‘state’ may be used uninitialized in this function" */
+            state = NETOPT_STATE_OFF;
```

Comment:
why not just initialize the variable with this value line 542 ?

Reply:
You're right. Will fix

Reply:
amended